lib/src/executables/dart2_upgrade.dart

```diff
@@ -98,20 +108,30 @@ void main(List<String> args) {
     phaseThreeSuggestors.add(PropsAndStateCompanionClassRemover());
   }
 
-  exitCode = runInteractiveCodemodSequence(
-    FileQuery.dir(
-      pathFilter: isDartFile,
-      recursive: true,
-    ),
-    [
-      AggregateSuggestor(
-        phaseTwoSuggestors.map((s) => Ignoreable(s)),
+  if (!args.contains(_helpFlag) && !args.contains('-h')) {
```

Comment:
Since we have a constant for the help flag, might as well have one for the abbreviation as well